llvm/utils/TableGen/Common/GlobalISel/GlobalISelMatchTable.h

```diff
@@ -492,9 +493,11 @@ class RuleMatcher : public Matcher {
   /// the renderers.
   StringMap<OperandMatcher *> DefinedOperands;
 
+  using PhysRegOperandsTy = MapVector<const Record *, OperandMatcher *>;
```

Comment:
---
Probably not a big deal:
```suggestion
  using PhysRegOperandsTy = SmallMapVector<const Record *, OperandMatcher *, 1>;
```